pkg/pipeline/extract/conntrack/conntrack.go

```diff
@@ -72,9 +72,9 @@ func (ct *conntrackImpl) Extract(flowLogs []config.GenericMap) []config.GenericM
 				Hash(computedHash).
 				KeysFrom(fl, ct.config.KeyDefinition).
 				Aggregators(ct.aggregators).
```

Comment:
KeysFrom(fl, ...) This seems to be dependent on the flow log. But this is used to define fields in the connType structure. Might the values not be different for the same connection with a different flow log?